### PR TITLE
Adding snippet and vocabulary: ui/ux

### PR DESCRIPTION
**Pull request by sannesofie**
### General Checks

- [x] the branch is up to date with `main`/`master`
- [x] the code works when pulled and run locally
- [x] All CI checks pass
- [x] all conflicts are resolved (if any)
- [x] PR has a descriptive title
- [x] PR has appropriate labels and milestones for easy identification
- [x] PR it is assigned to the owner
- [x] reviewers are assigned
- [x] the PR contributes only one focused change
- [x] It is in the appropriate column in the project board (if necessary)
- [x] has short and clear description
- [ ] is linked to an issue (if it is related)
- [x] feedback is addressed (if any and if it is appropriate feedback.)

### Markdown

- [x] the markdown source is formatted
- [x] spelling and grammar is correct in all text
- [x] The markdown looks correct when you preview the file
- [x] all links and images work
- [ ] any embedded code snippets are formatted

### CODE REVIEW

**Comment by arnochauveau** [comment]
Ah looks like you got past the linter this time, good job :-)

**Comment by sannesofie** [comment]
@arnochauveau Yes... very happy when things go my way :) Thanks for helping out

File: snippets/README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ naming conventions:
 - File extensions should show which language they are written in (except for
   english which is the default)
 - Files cannot be translated until there is an english version, so that everyone
-  can share this study guide
```
Comment (arnochauveau) [inline snippets/README.md:54]: I think this change happend by accident.

File: vocabulary/favicon-html.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# favicon
+
+## A favicon is a small image displayed next to the page title in the browser tab
+
+## It is a small image, so it should be a simple image with high contrast
+
+## To add a favicon to your website, either save your favicon image to the root directory of your webserver, or create a folder in the root directory called images, and save your favicon image in this folder. A common name for a favicon image is "favicon.ico" Next, add a `link` element to your "index.html" file, after the `title` element, like this
+
+<!DOCTYPE html>
+        <html>
```
Comment (arnochauveau) [inline vocabulary/favicon-html.md:10]: The html element is duplicated, which is invalid in HTML. 
Also consider indenting your code properly to make it more readable

File: snippets/html-css/contact-form.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+         <label for="subject">Subject</label>
+       <textarea id="subject" name="subject" placeholder="Write something.." style="height:200px"></textarea>
+
+    <input type="submit" value="Submit">
+
+  </form>
+</div>
+
+`CSS`
```
Comment (arnochauveau) [inline snippets/html-css/contact-form.md:29]: The correct syntax for code blocks in Markdown is ` ```css ` or ` ```html` to start and ` ``` ` to end your block

File: vocabulary/favicon-html.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# favicon
+
+## A favicon is a small image displayed next to the page title in the browser tab
+
+## It is a small image, so it should be a simple image with high contrast
+
+## To add a favicon to your website, either save your favicon image to the root directory of your webserver, or create a folder in the root directory called images, and save your favicon image in this folder. A common name for a favicon image is "favicon.ico" Next, add a `link` element to your "index.html" file, after the `title` element, like this
```
Comment (arnochauveau) [inline vocabulary/favicon-html.md:7]: `##` in md means secondary title, maybe consider normal text for the explanation ?

**Review by arnochauveau — CHANGES_REQUESTED** [review]
Good job! I've found some minor issues that you might wanna resolve

File: snippets/README.md (hunk 1 above)
Comment (sannesofie) [reply snippets/README.md:54]: Yes ;)

File: vocabulary/favicon-html.md (hunk 2 above)
Comment (sannesofie) [reply vocabulary/favicon-html.md:10]: Yes that was not meant to be duplicated.
Question: When doing html in VS code is there a way of formatting before pushing so I can be sure it is readable?

File: vocabulary/favicon-html.md (hunk 4 above)
Comment (sannesofie) [reply vocabulary/favicon-html.md:7]: Done

File: snippets/html-css/contact-form.md (hunk 3 above)
Comment (sannesofie) [reply snippets/html-css/contact-form.md:29]: Done

File: vocabulary/favicon-html.md (hunk 2 above)
Comment (ThibautHumblet) [reply vocabulary/favicon-html.md]: The VS Code has two explicit format actions: 
**Format Document** (Ctrl+Shift+I)
**Format Selection** (Ctrl+K Ctrl+F)

ALT+SHIFT+F works too if you select a formatter (the default works well, but you can also use ESLint for example)

Comment (sannesofie) [reply vocabulary/favicon-html.md]: @ThibautHumblet Awesome thanks for the tips ;)

**Comment by arnochauveau** [comment]
Issues weren't fixes 100%, I helped out a bit so you can check the commits to see what I meant :-)

**Review by arnochauveau — APPROVED** [review]

**Comment by sannesofie** [comment]
@arnochauveau I see the changes... I definitely need more practise but thanks it makes great sense when seeing the corrections

**Pull request merged**